src/makeCalculator.js

```diff
@@ -4,7 +4,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
+  const calculator = {
+    result: 0,
+  };
+
+  calculator.add = function (value) {
+    return this.result + value;
+  }.bind(calculator);
+
+  calculator.subtract = function (value) {
+    return this.result - value;
+  }.bind(calculator);
+
+  calculator.multiply = function (value) {
+    return this.result * value;
+  }.bind(calculator);
+
+  calculator.divide = function (value) {
+    return this.result / value;
+  }.bind(calculator);
```

Comment:
[CODE KNOWLEDGE] - Use the method shorthand syntax and assignment with arithmetic operator for the `divide` method. Additionally, ensure that division by zero is handled appropriately.

GOOD EXAMPLE:
divide(value) {
  if (value !== 0) {
    this.result /= value;
  }
},